src/containers/CreatePipelineRun/CreatePipelineRun.test.js

```diff
@@ -0,0 +1,404 @@
+};
+
+it('CreatePipelineRun renders', () => {
+  const { queryByText } = render(
+    <Provider store={testStoreAllNamespaces}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  expect(queryByText(/create pipelinerun/i)).toBeTruthy();
+});
+
+it('CreatePipelineRun renders controlled pipeline selection', () => {
+  const { queryByText } = render(
+    <Provider store={testStoreAllNamespaces}>
+      <CreatePipelineRun open pipelineName="pipeline-1" namespace="default" />
+    </Provider>
+  );
+  expect(queryByText(/create pipelinerun/i)).toBeTruthy();
+  expect(queryByText(/pipeline-1/i)).toBeTruthy();
+  expect(queryByText(/default/i)).toBeTruthy();
+});
+
+it('CreatePipelineRun renders namespace dropdown for selected namespace', () => {
+  const { queryByText } = render(
+    <Provider store={testStoreDefaultNamespace}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  expect(queryByText(/default/i)).toBeTruthy();
+});
+
+it('CreatePipelineRun resets pipeline and service account when namespace changes', () => {
+  const { getByText, getAllByText } = render(
+    <Provider store={testStoreDefaultNamespace}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  fillPipeline(getByText);
+  fillServiceAccount(getByText);
+  expect(getByText(/pipeline-1/i));
+  expect(getByText(/service-account-1/i));
+  fireEvent.click(getByText(/default/i));
+  fireEvent.click(getAllByText(/default/i)[1]);
+  expect(getByText(/pipeline-1/i));
+  expect(getByText(/service-account-1/i));
+});
+
+it('CreatePipelineRun validates form namespace', () => {
+  const scrollIntoViewMock = jest.fn();
+  const { getByText, queryByText } = render(
+    <Provider store={testStoreAllNamespaces}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  document.getElementById(
+    'create-pipelinerun--header'
+  ).scrollIntoView = scrollIntoViewMock;
+  fillServiceAccount(getByText);
+  fillPipeline(getByText);
+  fireEvent.click(getByText(/submit/i));
+  expect(queryByText(/unable to submit/i)).toBeTruthy();
+  expect(queryByText(/namespace cannot be empty/i)).toBeTruthy();
+});
+
+it('CreatePipelineRun validates form pipeline', () => {
+  const scrollIntoViewMock = jest.fn();
+  const { getByText, queryByText } = render(
+    <Provider store={testStoreAllNamespaces}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  document.getElementById(
+    'create-pipelinerun--header'
+  ).scrollIntoView = scrollIntoViewMock;
+  fillNamespace(getByText);
+  fillServiceAccount(getByText);
+  fireEvent.click(getByText(/submit/i));
+  expect(queryByText(/unable to submit/i)).toBeTruthy();
+  expect(queryByText(/pipeline cannot be empty/i)).toBeTruthy();
+});
+
+it('CreatePipelineRun validates form service account', () => {
+  const scrollIntoViewMock = jest.fn();
+  const { getByText, queryByText } = render(
+    <Provider store={testStoreAllNamespaces}>
+      <CreatePipelineRun open />
+    </Provider>
+  );
+  document.getElementById(
```

Comment:
you can get rid of all of these and their `const scrollIntoViewMock = jest.fn();`

Reply:
Sorry about that. I forgot to remove them.